code/nodes/optimizeModels/optimizeModels.q

```diff
@@ -26,11 +31,14 @@ optimizeModels.node.function:{[config;modelInfo;bestModel;modelName;tts;orderFun
   confMatrix:optimizeModels.confMatrix[hyperSearch`predictions;tts;config];
   impactReport:optimizeModels.impactDict[modelDict;hyperSearch;config];
   residuals:optimizeModels.residuals[hyperSearch;tts;config];
-  optimizeModels.consolidateParams[hyperSearch;confMatrix;impactReport;residuals] 
+  optimizeModels.consolidateParams[hyperSearch;confMatrix;impactReport;
+  residuals] 
```

Comment:
Should have double indentation for continuation of a line

Reply:
done